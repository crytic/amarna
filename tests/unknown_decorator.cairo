@foobar
func main():
    return ()

end
