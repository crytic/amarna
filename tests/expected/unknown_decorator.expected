[unknown-decorator] in unknown_decorator.cairo:1:2
[unused-function] in unknown_decorator.cairo:1:1